frontend/src/libs/components/header/header.tsx

```diff
@@ -18,12 +25,16 @@ const Header: React.FC = () => {
           <AppLogo />
         </Link>
         <div className={styles.navMenu}>
-          <Button
-            label="Sign In"
-            className={styles.btn}
-            type="button"
-            onClick={handleSignIn}
-          />
+          {hasUser ? (
+            <BurgerMenu />
```

Comment:
regarding which group is this user, we should throw burgerMenuItems here I think